src/components/RenderRouter/ContentItem.scss

```diff
@@ -258,6 +258,9 @@
         line-height: 7.5vw;
         margin-top: 13vw;
         display: inline-block;
+        border-bottom-width: 1px;
```

Comment:
I think this should be text-decoration, otherwise there are two underlines...

Reply:
@GeorgeBellTMH I had text decoration initially but the line is just too close to the word and we can't control the padding on the line with text decoration. Let me know how you guys would like to proceed.

Reply:
Sounds good...can you handle the hover state as well...

Reply:
(I know it's mobile, but it shows up if you have a small window on desktop as well)

Reply:
I'll work on the hover state as well. Do we have any guidelines for how it should appear on hover?